### DISCUSSION TITLE
Other Printers / custom

### DISCUSSION THREAD

**Discussion by lawhitecross** [mriscoc/Ender3V2S1#1151] [Ideas]
Morning, i am currently using your firmware on my current Ender 3v2 and is working like a charm.
We have an Wanhao D9 printer which we removed the mainboard and replaced with SKR BTT 1.4 turbo. The bed size is 400x400.
Is there anyway you could build me the firmware for this printer using your settings?

**Comment by lawhitecross** [comment]
or even you could get the firmware to work with SKR BTT 1.4 turbo with TMC2209 drivers and the TFT35 screen would be great

**Reply by mriscoc** [reply]
Hi, you need to try to compile the current Marlin bugfix for your printer, if you get a working firmware then the settings could be ported to the Professional firmware.

**Reply by propmok** [reply]
> or even you could get the firmware to work with SKR BTT 1.4 turbo with TMC2209 drivers and the TFT35 screen would be great

Did you ever get mriscoc on skr 1.4?

**Reply by lawhitecross** [reply]
No, i eventually decided to go full Klipper on everything. Easier to do changes as i needed without having to recompile every time and also i had a custom size printer
